src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -18,7 +18,7 @@
 * iot_support at tid dot es
 */
 
-package es.tid.fiware.perseo;
+package com.telefonica.iot.perseo;
```

Comment:
I understand that you are also solving https://github.com/telefonicaid/perseo-core/issues/5 in this PR. Great!

However, don't forget to mention in the CHANGES_NEXT_RELEASE file, e.g.

"Chaning the name of the Java package to com.telefonica.iot.perseo"

or something similar.